cmd/deployment/show.go

```diff
@@ -32,7 +35,10 @@ const showExample = `
   ecctl deployment show <deployment-id> --kind kibana
 
 * Shows apm resource information from a given deployment with a specified ref-id.
-  ecctl deployment show <deployment-id> --kind apm --ref-id apm-server`
+  ecctl deployment show <deployment-id> --kind apm --ref-id apm-server
```

Comment:
I mentioned this to Karen already, but the style guide at Elastic says to use `UPPERCASE_VARIABLE_NAMES_WITH_UNDERSCORES`, sometimes prefixed with `$`. I wouldn't change this here if `<variable-name>` is what you use in other places already, but it does put your docs out of sync with the ECE API reference for example (but even there we aren't always consistent).

Reply:
Ah, then we should fix it in the upcoming versions.